--- tools/reformat.sh
+++ tools/reformat.sh
@@ -54,8 +54,10 @@
 # Go
 $gofumpt -w -extra "$(pwd)"
 
-# Protobuf
-find . -name '*.proto' -exec "$clang_format" -i {} +
+# C++ and Protobuf
+find . \( -name '*.proto' -o -name '*.cc' -o -name '*.cpp' -o -name '*.cxx' \
+  -o -name '*.h' -o -name '*.hpp' -o -name '*.hxx' \) \
+  -exec "$clang_format" -i {} +
 
 # Generated .pb.go files
 find bazel-bin/ -path "*${go_module_name}*" -name '*.pb.go' -delete || true
